tests/improver-weighted-blending/08-nonlin_lin_args.bats

```diff
@@ -33,28 +33,27 @@
 
```

Comment:
Please could this test file be renamed to indicate that it is testing for an error.